rocky/reports/views/generate_report.py

```diff
@@ -65,30 +66,38 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
         )
 
 
-class OOISelectionGenerateReportView(BreadcrumbsGenerateReportView, BaseReportView, BaseOOIListView):
+class OOISelectionGenerateReportView(
+    BreadcrumbsGenerateReportView, BaseReportView, BaseOOIListView, KatGenerateReportStepsMixin
+):
     """
     Select objects for the 'Generate Report' flow.
     """
 
     template_name = "generate_report/select_oois.html"
-    current_step = 3
+    breadcrumbs_step = 3
+    current_step = 1
     ooi_types = get_ooi_types_with_report()
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["channel"] = "generate_report"
         context.update(self.get_ooi_filter_forms(self.ooi_types))
+        context["steps"] = self.build_steps()
+        context["current_step"] = self.get_current_step()
```

Comment:
Add this context to the main `BreadcrumbsGenerateReportView ` or `KatGenerateReportStepsMixin` so we do not repeat it in every view.